apps/admin/backend/src/app.ts

```diff
@@ -433,12 +434,21 @@ function buildApi({
       await logger.log(LogEventId.ImportCastVoteRecordsInit, userRole, {
         message: 'Importing cast vote records...',
       });
-      const importResult = await importCastVoteRecords(store, input.path);
+      const exportDirectoryPath =
+        // For manual export selection, users must select the contained metadata file as a proxy
+        // for the export directory since the UI doesn't support directory selection
```

Comment:
Closing the loop on https://github.com/votingworks/vxsuite/pull/3956#discussion_r1323644310!

Whereas you could previously select the top-level `cast-vote-record-report.json`, you can now select the top-level `metadata.json`. I couldn't find an easy way to support directory selection. Though non-ideal, we really only use this feature in dev. If an election official were to have to use it, I'd imagine it be in a context where we were guiding them.

Reply:
🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏 🙏